graphql/execute.lua

```diff
@@ -286,6 +286,36 @@ local function getFieldEntry(objectType, object, fields, context)
 
   arguments = setmetatable(arguments, {__index=positions})
 
+  local directiveMap = {}
+  for _, directive in ipairs(firstField.directives or {}) do
+    directiveMap[directive.name.value] = directive
+  end
+
+  local directives = {}
+
+  if next(directiveMap) ~= nil then
```

Comment:
Is it possible that directiveMap is nil?

You declare it as an empty table few lines above.

Reply:
It is possible that `directiveMap` is empty, if `firstField.directives` is nil or empty
https://github.com/tarantool/graphql/blob/a524ba756f1747a57b3d2335b3e74c7dbc55f262/graphql/execute.lua#L289-L290

Reply:
Oh. My bad. I've read a bit incorrect. Sorry.